server/routes/CheckServerRoutes.js

```
@@ -0,0 +1,68 @@
+const express = require('express');
+const app = express();
+const ServerPortRouter = express.Router();
+const checkPointCollectionDirectory = require('../models/CheckPointModel');
+
+ServerPortRouter.route('/add').post(function (req, res) {
+  
+  const checkPointBodyRequest = new checkPointCollectionDirectory(req.body);
+  checkPointBodyRequest.save()
+    .then(checkPointBodyRequest => {
+        res.json('Server added successfully');
+    })
+    .catch(err => {
+    res.status(400).send("unable to save to database");
+    });
+});
+
+ServerPortRouter.route('/').get(function (req, res) {
+    checkPointCollectionDirectory.find(function (err, serverports){
+    if(err){
+      console.log(err);
```

Return an error rather than log